geode-wan/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/AlterRegionCommandWithRemoteLocator.java

```diff
@@ -207,8 +207,34 @@ public void whenAlteringMultipleRegionWithAlterCommandToAddGatewaySendersThenItS
           "alter region --name=SecurityCrossReferences --gateway-sender-id=serialSender1,serialSender2");
       return true;
     });
+  }
 
+  /**
+   * Test execution of alter region command, when adding gw sender to already initialized region is
+   * not taking too much time.
+   * Verify that execution of command will not be halted, due to PRShadowRegion not available
+   * immediately in all servers.
+   */
+  @Test
+  public void alterInitializedRegionWithGwSenderOnManyServersDoesNotTakeTooLong() {
```

Comment:
This test case seems to pass in my laptop without the fix. Should more servers be added?

Reply:
Also, this is test that reproduce fault in old PR. You can repeat test case several times, it will fail.

Reply:
For sure, this would fail with changed timeout, but that cannot be changed. For details you can see comments in previous PR.